wordpress-mobile/WordPress-iOS
# Adds Fix All button action

Project: #15190
Related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/329

### To test:
1. Launch the app
2. Tap the My Site tab
3. Tap on a Jetpack Site with fixable threats
4. You should see a 'Fix All' button
5. Tap it
6. You should see a localized confirmation alert
7. Tap Cancel
8. Nothing should happen
9. Tap 'Fix All' Again
10. Tap on the 'Fix all threats' button
11. It will appear as if nothing happened
12. Refresh after a minute and there should be no fixable threats

### PR submission checklist:
